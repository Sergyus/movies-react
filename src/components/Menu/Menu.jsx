import React from 'react';
import styles from './Menu.scss';
import FilterPanelContainer from './components/FilterPanel';
import ChangeLayoutBtnsContainer from './components/ChangeLayoutBtns';

const Menu = () => (
  <section className={styles.menu} id="menu">
    <FilterPanelContainer />
    <ChangeLayoutBtnsContainer />
  </section>
);

export default Menu;
