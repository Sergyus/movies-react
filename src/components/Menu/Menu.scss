@import "../../styles/variables.scss";

.menu {
  display: flex;
  justify-content: space-between;
  border-bottom: 1px solid $gray-color;
  padding-top: 40px;
  margin-bottom: 30px;
  margin-left: 5%;
  margin-right: 5%;

  & > nav {
    flex-grow: 1;
  }
}

@media (max-width: $tablet-screen-size) {
  .menu {
    padding-top: 25px;
    border-bottom: none;
    margin-bottom: 10px;
  }

  .menu li {
    margin-bottom: 15px;
  }
}
