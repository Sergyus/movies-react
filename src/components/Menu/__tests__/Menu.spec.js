import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Menu from '..';

describe('Menu component renders correctly', () => {
  it('Menu renders correctly', () => {
    const renderer = new ShallowRenderer();
    const result = renderer.render(<Menu />);
    expect(result).toMatchSnapshot();
  });
});
