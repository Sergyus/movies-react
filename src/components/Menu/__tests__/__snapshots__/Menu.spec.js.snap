// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu component renders correctly Menu renders correctly 1`] = `
<section
  className="menu"
  id="menu"
>
  <Memo(ConnectFunction) />
  <Memo(ConnectFunction) />
</section>
`;
